### ISSUE TITLE
Using your code with the Arduino IDE

### ISSUE THREAD

**Issue by dynek** [eadf/esp8266_ping#3]
Hello,

Any chance to be able to use this code in the Arduino IDE - Have you tried so ?

Thank you

**Comment by eadf** [comment]
When I created this project I took inspiration from the [NewPing arduino library](https://bitbucket.org/teckel12/arduino-new-ping/wiki/Home). NewPing is interrupt based (accurate), supports maximum range (fast) and supports "single pin mode". Basically everything this project offers.

Now that the Arduino IDE can target esp8266 i think it's better to incorporate esp support into the existing arduino libraries than to create a lot of disparate arduino based, but still esp specific libraries.

tl;dr: no, use NewPing instead.

**Comment by dynek** [comment]
Noted thank you :-) The project I needed this for is "in production" already.
